docs/installation.md

```diff
@@ -94,10 +95,6 @@ be required if you installed SYCL in a non-standard location. See the [CMake
 documentation](https://cmake.org/documentation/) as well as the documentation
 for your SYCL implementation for more information on the other parameters.
 
-> We currently recommend using the [Ninja build
-> system](https://ninja-build.org/) for building hipSYCL-based projects due
-> to some issues with dependency tracking that CMake has with Unix Makefiles.
-
```

Comment:
Has this been resolved?

Reply:
Yes, that was purely a hipSYCL issue back when they had the `SYCL` language in CMake instead of the current compiler launcher integration.